bundle/edu.gemini.itc.shared/src/main/scala/edu/gemini/itc/shared/ConfigExtractor.scala

```diff
@@ -145,25 +142,15 @@ object ConfigExtractor {
   // TODO-GHOSTITC
   private def extractGhost(c: Config): String \/ GhostParameters = {
     // Gets the optional custom slit width
-    import GhostType._
-
-    def extractGain(r:ReadMode) : AmpGain = r match {
-      case GhostType.ReadMode.SLOW_LOW => GhostType.AmpGain.LOW
-      case GhostType.ReadMode.MEDIUM_LOW => GhostType.AmpGain.LOW
-      case GhostType.ReadMode.FAST_LOW => GhostType.AmpGain.LOW
-    }
 
+    // FIXME
     for {
-      readMode      <- extract[ReadMode]      (c, ReadModeKey)
-      specBin       <- extract[Binning]       (c, CcdXBinKey)
-      spatBin       <- extract[Binning]       (c, CcdYBinKey)
-      resolution    <- extract[Resolution]    (c, InsResolution)
-      nSkyMicrolens <- extract[Int]           (c, InsNskyMicrolens)
+      readMode      <- extract[GhostReadNoiseGain](c, ReadModeKey)
+      binning       <- extract[GhostBinning]      (c, CcdXBinKey)
```

Comment:
Do we still use the name `CcdXBinKey` in this case or should it be `CcdBinKey` or something?

Reply:
Good point